### PR TITLE
Updates Readme

### PR DESCRIPTION
**Pull request by Raahul-Singh**
Fixes pip install issues

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -34,7 +34,8 @@ install_requires =
     matplotlib
     torchvision
     scikit-learn
-    torchsde @ https://github.com/google-research/torchsde.git
+dependency_links=
+    https://github.com/google-research/torchsde.git
```
Comment (ucalyptus) [inline setup.cfg:38]: I think u need to add the git+  prefix here. 
As per https://www.freecodecamp.org/news/how-to-use-github-as-a-pypi-server-1c3b0d07db2/

Comment (Raahul-Singh) [reply setup.cfg:38]: Ooh I think I missed this. Thanks!

**Comment by Zymrael** [comment]
Closing for now as we're using a different `requirements` setup for workflows. I temporarily disabled tox tests, @Raahul-Singh could you take a look at those and use `OS-coverage` workflows as baseline on how to install dependencies (especially `torchsde`)?

**Comment by Raahul-Singh** [comment]
Hey! Sorry I somehow missed the notification. 
Sure! Let me get back to it!

**Pull request closed without merging**